types/dottie/index.d.ts

```diff
@@ -0,0 +1,194 @@
+// Type definitions for dottie 2.0
+// Project: https://github.com/mickhansen/dottie
+// Definitions by: Dom Armstrong <https://github.com/domarmstrong/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare const dottie: dottie.Dottie;
```

Comment:
Technically you could do `export default`, but since the intention doesn't seem to have been to make a default export, I guess it's better to leave it as is.